src/modules/PortAudio/PortAudio.cpp

```diff
@@ -88,15 +88,23 @@ ModuleSettingsWidget::ModuleSettingsWidget(Module &module) :
 	int idx = devicesB->findText(sets().getString("OutputDevice"));
 	devicesB->setCurrentIndex(idx < 0 ? 0 : idx);
 
+#ifdef Q_OS_MACOS
+	bitPerfect = new QCheckBox(tr("Bit-perfect audio"));
+	bitPerfect->setChecked(sets().getBool("BitPerfect"));
+	bitPerfect->setToolTip(tr("This sets the selected output device to the sample rate of the content being played"));
+#endif
+
 	QFormLayout *layout = new QFormLayout(this);
 	layout->addRow(enabledB);
 	layout->addRow(tr("Playback device") + ": ", devicesB);
 	layout->addRow(tr("Delay") + ": ", delayB);
+	layout->addRow(bitPerfect);
 }
 
 void ModuleSettingsWidget::saveSettings()
 {
 	sets().set("WriterEnabled", enabledB->isChecked());
 	sets().set("OutputDevice", devicesB->currentIndex() == 0 ? QString() : devicesB->currentText());
 	sets().set("Delay", delayB->value());
+	sets().set("BitPerfect", bitPerfect->isChecked());
```

Comment:
Crash, please add `#ifdef Q_OS_MACOS`

Reply:
oops, bis.

Shouldn't we in fact create the bitPerfect checkbox on all platforms but show it only on Mac? That would be the easiest way to ensure that the settings file preserves everything, even if you share it across different platforms.

Reply:
bitPerfect is available on Mac, so no need to store it on other platforms. If somebody adds it for Windows, then it will be changed.

Reply:
Missing `#ifdef Q_OS_MACOS` - it doesn't compile on Linux/Windows.